src/game_constants.h

```diff
@@ -73,6 +73,9 @@
 /** Temperature inside an active fridge in Fahrenheit  */
 #define FRIDGE_TEMPERATURE 37 // ~ 2.7 Celsius
 
+/** Temperature at which things are considered "cold" */
+#define COLD_TEMPERATURE 40
```

Comment:
I've previously been guided to use `static constexpr` instead of macros.  https://github.com/CleverRaven/Cataclysm-DDA/pull/23376#issuecomment-379059643

Reply:
The other temperature defines aren't constexpr's so making one that's a constexpr is weird.  I may consider changing them all to constexpr's but really someone should just wipe out all these defines into constexpr's.  Might be better as a separate PR

Reply:
Also making them `static` is 100% not the correct thing.